python/whylogs/api/writer/whylabs.py

```diff
@@ -177,6 +179,7 @@ def __init__(
         self._feature_weights = None
         self._reference_profile_name = config.get_whylabs_refernce_profile_name()
         self._api_config: Optional[Configuration] = None
+        self._prefer_sync = read_bool_env_var(WHYLOGS_PREFER_SYNC_KEY, False)
```

Comment:
Should this be `WHYLOGS_NO_ASYNC`? `prefer` kinda confuses me since it sounds like we might use sync, but we might use async as well.

Reply:
whylogs won't force a mode not exposed by whylabs-client, so the naming was trying to hint that this only affects those places where there is both an async/sync

Reply:
We could add support for WHYLOGS_NO_ASYNC naming and deprecate the "prefer" phrasing.

I'm inclined to keep the current value as we had some community testing with this naming already on the published dev build.